### DISCUSSION TITLE
Navigation Menu Component

### DISCUSSION THREAD

**Discussion by mrtzdev** [radix-ui/themes#553] [Ideas]
Is there a plan when the Navigation Menu component will be included in the theme release ? That would be really great if it can also be used with the radix theme styles. 

https://www.radix-ui.com/primitives/docs/components/navigation-menu

**Comment by FNDEVVE** [comment]
+1

**Comment by iamsean007** [comment]
+1. Though its easy to create your own with the current components too.
